src/pages/overview/index.md

```diff
@@ -51,7 +51,9 @@ The PDF Extract API provides a method for developers to extract and structure co
 
 The PDF Extract API can be embedded into any application using the [PDFServices SDK](./pdf-services-api/index.md#sdk) for Node.js, Python, and Java. [Start a PDF Extract free trial that includes up to 1,000 free Document transactions or runs for 6 months, whichever occurs earlier.](https://documentservices.adobe.com/dc-integration-creation-app-cdn/main.html?api=pdf-extract-api)
 
-## <span align="center" style="color:red;font-size:15px;vertical-align:middle;border:1px solid;padding-left:2px;padding-right:2px;">Available under Early Access Program</span><br/> PDF Accessibility Auto-Tag API for improving accessibility in PDFs
+## PDF Accessibility Auto-Tag API for improving accessibility in PDFs
+
+<span align="center" style="margin-top: 50px;color:red;font-size:15px;vertical-align:middle;border:1px solid;padding-left:2px;padding-right:2px;">Available under Early Access Program</span><br/>
```

Comment:
This is correct and do not put that <span> inside the header.  You just need to add display:inline-block and change the margin-top: 10px in the style.  Like this <span align="center" style="display:inline-block;margin-top:10px;color:red;font-size:15px;vertical-align:middle;border:1px solid;padding-left:2px;padding-right:2px;">Available under Early Access Program</span><br/>

Reply:
Done